core/src/main/java/com/sequenceiq/cloudbreak/service/cluster/clouderamanager/ClouderaManagerModificationService.java

```diff
@@ -80,6 +82,53 @@ public void stopCluster(Stack stack) throws CloudbreakException {
 
     @Override
     public int startCluster(Stack stack) throws CloudbreakException {
```

Comment:
Do we have unit test for this method? If not, please write.

Reply:
We're covering this with IT mock tests.